lib/rules/jest/no-vague-titles.js

```diff
@@ -0,0 +1,92 @@
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Prevent the usage of vague words in test statements.',
+      category: 'Best Practices',
+      recommended: false,
+      uri:
+        'https://github.com/Shopify/eslint-plugin-shopify/blob/master/docs/rules/no-vague-titles.md',
+    },
+  },
+  create(context) {
+    const ignored = (context.options[0] && context.options[0].ignore) || [];
+
+    function isIgnoredFunctionName(node) {
+      if (node.type === 'MemberExpression') {
+        return ignored.some((method) => method === node.object.name);
+      }
+      return ignored.some((method) => method === node.callee.name);
+    }
+
+    function validate(node) {
+      if (
+        !isTestFunction(node) ||
+        isIgnoredFunctionName(node) ||
+        !containsDescription(node)
+      ) {
+        return;
+      }
+
+      const description = getDescription(node);
+
+      if (containsVagueWord(description)) {
+        context.report({
+          message: `{{ method }} statement should not contain vague words`,
+          data: {method: node.callee ? node.callee.name : node.object.name},
+          node,
+        });
+      }
+    }
+
+    return {
+      MemberExpression(node) {
```

Comment:
Why do you need this one? I think we only care about call expressions.

Reply:
@lemonmade `it.only`

Reply:
But `validate` always works for call expressions, regardless of whether the callee is just an identifier or a member expression. You end up basically validating every member expression twice (once at the call expression, once at the member expression)